apps/taxes/src/modules/taxes/providers/taxjar/taxjar-provider.ts

```diff
@@ -0,0 +1,45 @@
+import { TaxProviderName } from "..";
+import { TaxBaseFragment } from "../../../../../generated/graphql";
+import { logger } from "../../../../lib/logger";
+import {
+  ChannelConfig,
+  defaultChannelConfig,
+} from "../../../channels-configuration/channels-config";
+import { TaxProvider } from "../../tax-provider";
+import { taxJarCalculate } from "./taxjar-calculate";
+import { TaxJarClient } from "./taxjar-client";
+import { defaultTaxJarConfig, TaxJarConfig } from "./taxjar-config";
+
+export class TaxJarProvider implements TaxProvider {
+  name = "taxjar" as TaxProviderName;
```

Comment:
should be enough to write `readonly name = "taxjar"`, then its narrowed down and immutable. 

`as` assertion is always less secure (if you make a typo with value, TS will allow it)

also if fields only can be readonly, they should be